packages/bitcore-client/bin/wallet-create-lite

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
```

Comment:
curious why a separate script over adding the --lite flag

Reply:
I could do that, I don't think there was a particular reason